scenes/src/app/scenes/progress-spinner/progress-spinner-scene.scss

```diff
@@ -0,0 +1,12 @@
+mat-progress-spinner {
+  position: absolute;
```

Comment:
Is it possible to use flexbox or some other alignment strategy to get this perfectly centered rather than trying to calculate it?